File: Shopping/ShoppingCommand/src/main/java/nl/avans/domain/models/events/product/ProductCreatedEvent.java
package nl.avans.domain.models.events.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import nl.avans.domain.models.events.Event;
import nl.avans.domain.models.models.Product;

import java.util.UUID;

public class ProductCreatedEvent implements Event {
    @JsonProperty
    private UUID productId;

    @JsonProperty
    private String name;

    @JsonProperty
    private int amount;

    @JsonProperty
    private String description;

    @JsonProperty
    private double price;

    @JsonProperty
    private double weight;

    @JsonProperty
    private String brandName;

    @JsonProperty
    private String supplierName;
}

File: InventoryQueries/Domain/Events/ProductCreatedEvent.cs
using System;

namespace Domain.Events
{
	public class ProductCreatedEvent
	{
		public ProductCreatedEvent() { }

		public Guid ProductId { get; set; }
		public string Name { get; set; }
		public string BarCode { get; set; }
		public int Amount { get; set; }
		public string Description { get; set; }
		public decimal Price { get; set; }
		public string BrandName { get; set; }
		public string SupplierName { get; set; } = "Ball.com";
		public decimal Weight { get; set; }
	}
}
